lace/lace_cc/src/state.rs

```diff
@@ -559,10 +551,9 @@ impl State {
         let mut col_ixs: Vec<usize> = (0..self.n_cols()).collect();
         col_ixs.shuffle(rng);
 
-        self.loglike = col_ixs
-            .drain(..)
-            .map(|col_ix| self.reassign_col_gibbs(col_ix, draw_alpha, rng))
-            .sum::<f64>();
+        col_ixs.drain(..).for_each(|col_ix| {
+            self.reassign_col_gibbs(col_ix, draw_alpha, rng);
+        })
```

Comment:
Do we not need to update `self.loglike` anymore?

Reply:
no. it gets updated by `step` .

https://github.com/promised-ai/lace/blob/c7300686d8a2eb6cc726bd2a0ded61d87533e0e3/lace/lace_cc/src/state.rs#L242